components/build-service/base/rbac/build-config.yaml

```diff
@@ -0,0 +1,26 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: build-pipeline-config-read-only
+  namespace: build-service
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  resourceNames: ["build-pipeline-config"]
+  verbs: ["get"]
```

Comment:
Maybe this file should be in the build-service repo? That's where the BuildPipelineSelector role lives too https://github.com/konflux-ci/build-service/blob/main/config/rbac/buildpipelineselector_viewer_role.yaml

Reply:
Yes, let's move it into Build Service, since the role might be needed for community edition of Konflux.

Reply:
moved role to build service